ports/rp2/rp2_pio.c

```diff
@@ -422,13 +422,14 @@ STATIC void rp2_state_machine_print(const mp_print_t *print, mp_obj_t self_in, m
 }
 
 // StateMachine.init(prog, freq=-1, *,
-//     in_base=None, out_base=None, set_base=None, sideset_base=None,
-//     in_shiftdir=None, out_shiftdir=None, push_thresh=None, pull_thresh=None,
+//     in_base=None, out_base=None, set_base=None, jmp_pin=None,
+//     sideset_base=None, in_shiftdir=None, out_shiftdir=None,
+//     push_thresh=None, pull_thresh=None,
 // )
 STATIC mp_obj_t rp2_state_machine_init_helper(const rp2_state_machine_obj_t *self, size_t n_args, const mp_obj_t *pos_args, mp_map_t *kw_args) {
     enum {
         ARG_prog, ARG_freq,
-        ARG_in_base, ARG_out_base, ARG_set_base, ARG_sideset_base,
+        ARG_in_base, ARG_out_base, ARG_set_base, ARG_jmp_pin, ARG_sideset_base,
```

Comment:
I don't know if `jmp_pin` is the best name, should it be `jump_base` for example?

Reply:
I think `jmp_pin` is a decent choice.  It should definitely start with "jmp" to match the instruction name.  And I think using "pin" is clearer than "base" because it doesn't support a range of pins, only one.

Reply:
Sounds good to me!